doc/cephfs/cephfs-mirroring.rst

```diff
@@ -301,6 +301,63 @@ synchronization.
 When mirroring is disabled, the respective `fs mirror status` command for the file system
 will not show up in command help.
 
+Configuration Options
+---------------------
+
+``cephfs_mirror_max_concurrent_directory_syncs``
+
+:Description:  Maximum number of concurrent snapshot synchronization threads
+:Type:  64-bit Integer Unsigned
+:Default: ``3``
```

Comment:
instead of repeating the description, type and default value in the document, i'd suggest use
```rst
.. confval:: cephfs_mirror_max_concurrent_directory_syncs
```

see https://github.com/ceph/ceph/blob/master/doc/rados/configuration/auth-config-ref.rst#enablement for an example.

the `.. confval::` directive is able to lookup the option by name and extract the interesting bits out of `src/common/options/cephfs-mirror.yaml.in`. see https://docs.ceph.com/en/latest/rados/configuration/auth-config-ref/#confval_option-auth_cluster_required for the generated document.

Reply:
nice, thanks @tchaikov

Reply:
That's really awesome. Great work @tchaikov !